app/javascript/angular/code/values/session.js

```diff
@@ -34,6 +34,13 @@ export const startingLatLng = (session, selectedSensor) => {
   };
 };
 
+export const startingLatLngLiteral = (session, selectedSensor) => {
+  return {
+    lat: startingLat(session, selectedSensor),
+    lng: startingLng(session, selectedSensor)
+  };
+};
+
```

Comment:
if the function only `return`s something you can write it like this:

```
export const startingLatLngLiteral = (session, selectedSensor) => ({
  lat: startingLat(session, selectedSensor),
  lng: startingLng(session, selectedSensor)
});
```
(or however you prefer to put the {}, honestly I don't even remember now what I usually do :D)